lib/index.js

```diff
@@ -82,14 +87,16 @@ Optimizely.prototype.initialize = function() {
  */
 
 Optimizely.prototype.track = function(track) {
-  var payload = {};
+  var attributes = track.properties();
+  var payload = {
+    'tags': attributes || {},
```

Comment:
track.properties() returns an empty {} if no properties were passed so this fallback is unnecessary :)

could we also remove the quotes around` 'tags'`?